src/test/resources/application.conf

```diff
@@ -0,0 +1,36 @@
+// a test DB is required.  Here's an example command to run one locally
```

Comment:
IIRC there's some shenanigans that happens when you have an application.conf provided by your application and then someone else wants to provide their own conf on top. We use reference.conf in rawls (and other services) for this reason.